client/my-sites/email/inbox/index.jsx

```diff
@@ -18,6 +18,8 @@ import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 
 import './style.scss';
 
+export const INBOX = 'inbox';
```

Comment:
I find it difficult to understand how we should use that constant, or what is it for (e.g. when I see this I wonder if `inbox` has not been localized on purpose). Maybe we could add a prefix to clarify that?

```suggestion
export const SOURCE_INBOX = 'inbox';
```

Should we also move it to a `constants.js` file, or do we prefer to keep it here?